### PR TITLE
Orhan-week-5

### PR DESCRIPTION
**Pull request by OBakir90**
# Your Details

Your Name:
Your City:
Your Slack Name:

# Homework Details

Module:
Week:

### CODE REVIEW

File: week-5/Homework/mandatory/2-exercises/exercises.js [hunk 1]
```diff
@@ -39,6 +39,14 @@
...
   let content = document.querySelector("#content");
+  arrayOfPeople.forEach(p => {
+    let h1 = document.createElement('h1')
+    h1.innerText = p.name
+    let h2 = document.createElement('h2')
+    h2.innerText = p.job
+    content.appendChild(h1)
+    content.appendChild(h2)
+  })
   //Write your code in here
```
Comment (Ashaghel) [inline week-5/Homework/mandatory/2-exercises/exercises.js:50]: Perfect work here

File: week-5/Homework/mandatory/2-exercises/exercises.js [hunk 2]
```diff
@@ -52,7 +60,13 @@ function insertPeopleData(arrayOfPeople) {
...
 function insertShoppingList(shoppingList) {
-  //Write your code in here
+  let unorderedList = document.createElement('ul')
+  shoppingList.forEach(p => {
+    let listItem = document.createElement('li')
+    listItem.innerText = p
+    unorderedList.appendChild(listItem)
+  })
+  content.appendChild(unorderedList)
 }
```
Comment (Ashaghel) [inline week-5/Homework/mandatory/2-exercises/exercises.js:70]: Same here 🥇

File: week-5/Homework/mandatory/2-exercises/exercises.js [hunk 3]
```diff
@@ -72,7 +86,26 @@ function insertShoppingList(shoppingList) {
...
+    let bookName = document.createElement('p')
+    bookName.innerText = `${p.title}-${p.author}`
+    let bookImg = document.createElement('img')
+    bookImg.style.width = '200px'
+    bookImg.style.margin = '5px'
+    bookImg.src = bookImgSrc[i]
+    book.appendChild(bookName)
+    book.appendChild(bookImg)
+    bookslist.appendChild(book)
+  })
```
Comment (Ashaghel) [inline week-5/Homework/mandatory/2-exercises/exercises.js:107]: Perfect again
how about adding the link to the image (url maybe)
more image attributes (alt, href, target)

File: week-5/Homework/mandatory/3-project/js/main.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    donateBtn.style.backgroundColor = "#5751fd"
+    volunteerBtn.style.backgroundColor = "#31b0d5"
+    volunteerBtn.style.color = "white"
+})
+
+greenBtn.addEventListener('click', () => {
+    jumbotron.style.backgroundColor = "#87ca8a"
+    donateBtn.style.backgroundColor = "black"
+    volunteerBtn.style.backgroundColor = "#8c9c08"
+})
```
Comment (Ashaghel) [inline week-5/Homework/mandatory/3-project/js/main.js:29]: looks good
there is a room for improvement 
all event listner do same thing here
maybe change color to array of object (or array only)?

File: week-5/Homework/mandatory/3-project/js/main.js [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+const formControl = document.querySelectorAll('.form-control')
+const submitBtn = document.querySelector('form button')
+
+// Solution 1
+
+// function submitForm(e) {
+//     e.preventDefault()
+//     for (let i = 0; i < formControl.length; i++) {
+//         if (formControl[i].value.length > 0) {
+//             formControl[0].value.includes('@') ? alert('thank you for filling out the form') : alert('please check your e-mail adresse')
```
Comment (Ashaghel) [inline week-5/Homework/mandatory/3-project/js/main.js:41]: smart solution, but best you specify the email one at least.

File: week-5/Homework/mandatory/3-project/js/main.js [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+// Solution 2
+
+function submitForm(e) {
+    e.preventDefault()
+    formControl.forEach(p => {
+        if (p.value.length > 0) {
+            p.type === 'email' && p.value.includes('@') ? alert('thank you for filling out the form') : alert('please check your e-mail adresse')
+        } else {
+            alert('please fill required places in the form')
+        }
```
Comment (Ashaghel) [inline week-5/Homework/mandatory/3-project/js/main.js:56]: something goes wrong here ..you see the alert far too many times !!!
also if you have something not correct, the alert will remove all other correct form inputs!

Comment (OBakir90) [reply week-5/Homework/mandatory/3-project/js/main.js:52]: fixed code

**Pull request closed without merging**